{
  "_annotation": {
    "cve_id": "CVE-2024-32880",
    "reason": "Added CPE configurations because not yet analyzed by NVD.",
    "snapshot": "https://raw.githubusercontent.com/anchore/nvd-data-overrides/main/.snapshot/2024/CVE-2024-32880.json",
    "triage_notes": {
      "references": [
        "https://github.com/pyload/pyload/security/advisories/GHSA-3f7w-p8vr-4v5f"
      ]
    }
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:pyload-ng_project:pyload-ng:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "CC928D5C-FE46-5D86-B24D-66497670B316",
                "vulnerable": true
              },
              {
                "criteria": "cpe:2.3:a:pyload:pyload:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "A0E21CC1-8EA2-5BCF-92A5-3EF25054E629",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}